### PR TITLE
gcp variable names consistency

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

File: gcp/modules/dcos-tested-gcp-oses/input_test_profile [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-# azure_sku = 7.3
-# azure_version = 7.3.2017053118
+# gcp_offer = RHEL
+# gcp_publisher = RedHat
+# gcp_sku = 7.3
+# gcp_version = 7.3.2017053118
 # ------------------------------
-provider = "azure"
+provider = "gcp"
 os = "rhel_7.3"
```
Comment (pires) [inline gcp/modules/dcos-tested-gcp-oses/input_test_profile]: Wait, what is this? How can we select the OS we want? It's very important to the Kubernetes team to run CentOS 7.4, CoreOS (stable) and Ubuntu LTS in different, parallel test runs.

File: gcp/modules/dcos-tested-gcp-oses/input_test_profile [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 #os = "centos_7.3"
 #os = "coreos_835.13.0"
 #os = "coreos_1235.9.0"
-region = "westus"
+region = "us-west1"
```
Comment (pires) [inline gcp/modules/dcos-tested-gcp-oses/input_test_profile]: How can we change the region? Are these variables too?

File: gcp/modules/dcos-tested-gcp-oses/input_test_profile (hunk 1 above)
Comment (rimusz) [reply gcp/modules/dcos-tested-gcp-oses/input_test_profile]: yup

File: gcp/modules/dcos-tested-gcp-oses/input_test_profile (hunk 2 above)
Comment (rimusz) [reply gcp/modules/dcos-tested-gcp-oses/input_test_profile]: yup, all in `desired_cluster_provile`

**Pull request merged**